src/containers/shared/components/Transaction/AMMVote/test/AMMVote.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { Simple } from '../Simple'
+import { createSimpleWrapperFactory, expectSimpleRowText } from '../../test'
+import voteMock from './mock_data/amm_vote.json'
+
+describe('AMM Vote Tests', () => {
+  const createWrapper = createSimpleWrapperFactory(Simple)
+
+  it('renders from transaction', () => {
+    const wrapper = createWrapper(voteMock)
+    expectSimpleRowText(wrapper, 'trading_fee', '%0.001')
```

Comment:
Shouldn't the percent symbol `%` be the suffix? For example, `0.001%`

Reply:
done